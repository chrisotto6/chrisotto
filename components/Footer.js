import Link from 'next/link'
import siteMetadata from '@/data/siteMetadata'
import { FaEnvelope, FaGithub, FaLinkedin, FaTwitter } from 'react-icons/fa'

export default function Footer() {
  return (
    <footer>
      <div className="mt-16 flex flex-col items-center">
        <div className="mb-5 flex space-x-2 text-sm text-gray-500 dark:text-gray-400">
          <Link href="/">{siteMetadata.title}</Link>
          <div>{` • `}</div>
          <div>{`© ${new Date().getFullYear()}`}</div>
        </div>
        <div className="mb-3 flex space-x-8">
          <Link href={`mailto:${siteMetadata.email}`}>
            <FaEnvelope size={30} />
          </Link>
          <Link href={siteMetadata.github}>
            <FaGithub size={30} />
          </Link>
          <Link href={siteMetadata.linkedin}>
            <FaLinkedin size={30} />
          </Link>
          <Link href={siteMetadata.twitter}>
            <FaTwitter size={30} />
          </Link>
        </div>
      </div>
    </footer>
  )
}
