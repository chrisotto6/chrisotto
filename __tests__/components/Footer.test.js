import { render } from '@testing-library/react'
import Footer from '@/components/Footer'

it('renders Footer unchanged', () => {
  const { container } = render(<Footer />)
  expect(container).toMatchSnapshot()
})
