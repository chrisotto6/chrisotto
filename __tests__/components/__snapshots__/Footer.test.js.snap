// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Footer unchanged 1`] = `
<div>
  <footer>
    <div
      class="mt-16 flex flex-col items-center"
    >
      <div
        class="mb-3 flex space-x-4"
      >
        <a
          class="text-sm text-gray-500 transition hover:text-gray-600"
          href="mailto:chris@chrisotto.dev"
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            class="sr-only"
          >
            mail
          </span>
          <test-file-stub
            classname="fill-current text-gray-700 hover:text-blue-500 dark:text-gray-200 dark:hover:text-blue-400 h-6 w-6"
          />
        </a>
        <a
          class="text-sm text-gray-500 transition hover:text-gray-600"
          href="https://github.com/chrisotto6"
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            class="sr-only"
          >
            github
          </span>
          <test-file-stub
            classname="fill-current text-gray-700 hover:text-blue-500 dark:text-gray-200 dark:hover:text-blue-400 h-6 w-6"
          />
        </a>
        <a
          class="text-sm text-gray-500 transition hover:text-gray-600"
          href="https://www.linkedin.com/in/ottochristopher/"
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            class="sr-only"
          >
            linkedin
          </span>
          <test-file-stub
            classname="fill-current text-gray-700 hover:text-blue-500 dark:text-gray-200 dark:hover:text-blue-400 h-6 w-6"
          />
        </a>
        <a
          class="text-sm text-gray-500 transition hover:text-gray-600"
          href="https://twitter.com/chris_otto6"
          rel="noopener noreferrer"
          target="_blank"
        >
          <span
            class="sr-only"
          >
            twitter
          </span>
          <test-file-stub
            classname="fill-current text-gray-700 hover:text-blue-500 dark:text-gray-200 dark:hover:text-blue-400 h-6 w-6"
          />
        </a>
      </div>
      <div
        class="mb-5 flex space-x-2 text-sm text-gray-500 dark:text-gray-400"
      >
        <div>
          Chris Otto
        </div>
        <div>
           • 
        </div>
        <div>
          © 2022
        </div>
        <div>
           • 
        </div>
        <a
          href="/"
        >
          Chris Otto | Father, Developer, Test Engineer
        </a>
      </div>
    </div>
  </footer>
</div>
`;
